### ISSUE TITLE
Switch default state store to SQLite for local experience

### ISSUE THREAD

**Issue by ItalyPaleAle** [dapr/cli#1328]
## Describe the proposal

Running `dapr init` sets up a default state store "statestore" that is based on Redis. Redis has some issues when used as actor state store, so we should start by migrating the default state store used in the local experience to SQLite.

## Release Note

RELEASE NOTE: **UPDATE** Switch default state store to SQLite for local experience

**Comment by mukundansundar** [comment]
This requires a config file change.

**Comment by ItalyPaleAle** [comment]
Is there any update on this?

**Comment by mukundansundar** [comment]
As discussed in an endgame sync, moving this to next milestone for now.

**Comment by oleksandr-bilyk** [comment]
Yes please also update documentation how to do it withy VS2022 + docker-compose + .NET
